src/KITT.Web.App.UI/Components/ConfirmDialog.razor

```diff
@@ -12,17 +12,17 @@
 
 @code {
 	[CascadingParameter]
-	MudDialogInstance Dialog { get; set; }
+	MudDialogInstance? Dialog { get; set; }
```

Comment:
Maybe I'll leave this instance not nullable but I will initialize with default!:
```
MudDialogInstance Dialog { get; set; } = default!;
```